### DISCUSSION TITLE
Integration with Laravel Jetstream Auth

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [spatie/laravel-honeypot#98] [Q&A]
How do I go about integrating this with Laravel Jetstream Auth? I get some spam user registrations and would like to restrict it using this package.

**Comment by abishekrsrikaanth** [comment]
@freekmurze do you have any suggestions for this?

**Reply by kurucu** [reply]
The readme has now been updated with Livewire and Inertia examples, which should cover you off. Inertia seems like a lot more hard work, as you have to add the form to all Fortify forms, rather than just registration.
